health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -77,74 +77,169 @@ export const UICustomizations = {
-                }
+              setShowToast({ label: `No file with templateIdentifier '${template}' found` });
+              return;
+            }
             const campaignName = row?.name || "";
+            const customName = type === "Estimations" ? campaignName : `${campaignName}_DRAFT`;
             Digit.Utils.campaign.downloadExcelWithCustomName({
               fileStoreId: fileId,
-              customName: campaignName
+              customName: customName,
             });
           };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)